src/rgw/rgw_main.cc

```diff
@@ -689,6 +697,9 @@ int radosgw_Main(int argc, const char **argv)
 #ifdef WITH_RADOSGW_KAFKA_ENDPOINT
   rgw::kafka::shutdown();
 #endif
+#ifdef WITH_RADOSGW_LUA_PACKAGES
```

Comment:
this should not be under the lua packages ifdef
this ifdef is only used to disable luarocks support where we don't want it